packages/client/src/lib/app-websocket.ts

```
@@ -155,6 +158,10 @@ export function connect(): void {
       setState({ connected: false, sessionsSynced: false, agentsSynced: false, repositoriesSynced: false });
       logger.debug(`[WebSocket] Disconnected (code: ${event.code}, reason: ${event.reason || 'none'})`);
 
+      if (event.code === WS_CLOSE_CODE.POLICY_VIOLATION) {
+        policyViolationListeners.forEach(fn => fn());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)